posthog/migrations/0867_initial_migration.py

```diff
@@ -0,0 +1,195 @@
+# Generated by Django 4.2.22 on 2025-09-29 09:03
+
+from django.db import migrations
+
+
+def update_content_type(apps, schema_editor):
+    ContentType = apps.get_model("contenttypes", "ContentType")
+    for model in [
+        "errortrackingassignmentrule",
+        "errortrackingexternalreference",
+        "errortrackinggroup",
+        "errortrackinggroupingrule",
+        "errortrackingissue",
+        "errortrackingissueassignment",
+        "errortrackingissuefingerprint",
+        "errortrackingissuefingerprintv2",
+        "errortrackingrelease",
+        "errortrackingstackframe",
+        "errortrackingsuppressionrule",
+        "errortrackingsymbolset",
+    ]:
+        try:
+            ct = ContentType.objects.get(app_label="posthog", model=model)
+            ct.app_label = "error_tracking"
+            ct.save()
+        except ContentType.DoesNotExist:
+            pass
+
+
+def reverse_content_type(apps, schema_editor):
+    ContentType = apps.get_model("contenttypes", "ContentType")
+    for model in [
+        "errortrackingassignmentrule",
+        "errortrackingexternalreference",
+        "errortrackinggroup",
+        "errortrackinggroupingrule",
+        "errortrackingissue",
+        "errortrackingissueassignment",
+        "errortrackingissuefingerprint",
+        "errortrackingissuefingerprintv2",
+        "errortrackingrelease",
+        "errortrackingstackframe",
+        "errortrackingsuppressionrule",
+        "errortrackingsymbolset",
+    ]:
+        try:
+            ct = ContentType.objects.get(app_label="error_tracking", model=model)
+            ct.app_label = "posthog"
+            ct.save()
+        except ContentType.DoesNotExist:
+            pass
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("posthog", "0866_add_integration_to_batch_export_destination"),
+    ]
+
+    operations = [
+        migrations.SeparateDatabaseAndState(
+            database_operations=[
+                migrations.RunPython(update_content_type, reverse_content_type),
```

Comment:
I don't really understand what this does / why it's necessary but looks reasonable